### ISSUE TITLE
Mount HDFS broken against latest cluster

### ISSUE THREAD

**Issue by kevcunnane** [microsoft/azuredatastudio#8159]
Issue Type: <b>Bug</b>

When trying to mount we get an error, looks likely to be an API issue:

Error mounting folder: Error creating mount - Required parameter remote was null or undefined when calling createMount.

Azure Data Studio version: azuredatastudio Dev 1.14.0 (9ca6200018fc206d67a47229f991901a8a453781, 2017-12-15T12:00:00.000Z)
OS version: Darwin x64 18.7.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz (12 x 2600)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off<br>surface_control: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 2, 2|
|Memory (System)|16.00GB (0.07GB free)|
|Process Argv|.|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (3)</summary>

Extension|Author (truncated)|Version
---|---|---
gitlens|eam|9.9.3
ads-language-pack-es|Mic|1.9.0
azuredatastudio-postgresql|Mic|0.2.2

</details>

**Comment by kevcunnane** [comment]
Simple fix and impacts a key SQL2019 scenario. Approved for merge to release once it passes CI & is merged into master
